### ISSUE TITLE
doesn't seem to work on https links in markup

### ISSUE THREAD

**Issue by binarykitchen** [patrick96/click-link#4]
for example

```
1. Install [Chrome Dev for Linux](https://www.google.com/chrome/browser/?platform=linux&extra=devchannel)
```

ctrl-click on that google link doesn't do anything

**Comment by patrick96** [comment]
Thanks for reporting. I don't seem to be able to reproduce this issue. Could you give me a little more information:

1. Which version under what OS are you running?
2. Do only links in markdown not work or does this happen in other places too?
3. What other packages have you installed?

Could you also try to only load this package as well as [language-hyperlink](https://atom.io/packages/language-hyperlink) in atom and see if the problem persists

**Comment by binarykitchen** [comment]
Ubuntu 16.10 here, running Atom v1.12.6 and Click-Link v1.0.5

It's only these kind of links, others do work fine form.

Other packages? Click-Link, Lisp-Paredit, Project-Manager and Proto-Repl

**Comment by patrick96** [comment]
Thank you. I have been able to reproduce this now, just had to actually change the file type to markup.

It seems language-hyperlink does not set hyperlink class names in markdown. I will look at it more closely in a few hours but this isn't a simple fix.

**Comment by patrick96** [comment]
With the new version I can no longer reproduce this issue, so I hope it's fixed on your end too

**Comment by binarykitchen** [comment]
well, done, it's fixed now. thx man!

**Comment by patrick96** [comment]
I have just reverted the fix for this issue and replaced it with a solution that's not as good but should still get the job done.
The original fix was causing some issues (see #6) because it added a new dependency.

If you experience any problems with links in markdown fell free to reopen this issue.
